content/learn/book/migration-guides/0.9-0.10/_index.md

```diff
@@ -38,9 +38,245 @@ As a result, the Minimum Supported Rust Version (MSRV) is "the latest stable rel
+```
+
+If you had your own stage:
+
+```rust
+// Bevy 0.9
+#[derive(Debug, Hash, PartialEq, Eq, Clone, StageLabel)]
+pub struct AfterUpdate;
+
+app.add_stage_after(CoreStage::Update, AfterUpdate, SystemStage::parallel());
+
+// Bevy 0.10, no command flush
+#[derive(Debug, Hash, PartialEq, Eq, Clone, SystemSet)]
+#[system_set(base)]
+pub struct AfterUpdate;
+
+app.configure_set(
+    AfterUpdate
+        .after(CoreSet::UpdateFlush)
+        .before(CoreSet::PostUpdate),
+);
+
+// Bevy 0.10, with a command flush
+#[derive(Debug, Hash, PartialEq, Eq, Clone, SystemSet)]
+#[system_set(base)]
+pub enum AfterUpdate {
+    Parallel,
+    CommandFlush
+}
+
+app.configure_sets(
+    (
+        CoreSet::UpdateFlush,
+        AfterUpdate::Parallel,
+        AfterUpdate::CommandFlush,
+        CoreSet::PostUpdate,
+    ).chain()
+).add_system(apply_system_buffers.in_base_set(AfterUpdate::CommandFlush));
+
+```
+
+#### Label types
+
+System labels have been renamed to systems sets and unified with stage labels.
+The `StageLabel` trait should be replaced by a system set, using the `SystemSet` trait as dicussed immediately below.
+
+Before:
+
+```rust
+#[derive(Debug, Hash, PartialEq, Eq, Clone, StageLabel)]
+enum MyStage {
+    BeforeRound,
+    AfterRound,
+}
+
+#[derive(Debug, Hash, PartialEq, Eq, Clone, SystemLabel)]
+enum MySystem {
+    ComputeForces,
+    FindCollisions,
+}
+```
+
+After:
+
+```rust
+#[derive(Debug, Hash, PartialEq, Eq, Clone, SystemSet)]
+enum MySet {
+    BeforeRound,
+    AfterRound,
+}
+
+#[derive(Debug, Hash, PartialEq, Eq, Clone, SystemSet)]
+enum MySystem {
+    ComputeForces,
+    FindCollisions,
+}
+```
+
+#### System sets (Bevy 0.9)
+
+In Bevy 0.9, you could use the `SystemSet` type and various methods to configure many systems at once.
+Additionally, this was the _only_ way to interact with various scheduling APIs like run criteria.
+
+Before:
+
+```rust
+app.add_system_set(SystemSet::new().with_system(a).with_system(b).with_run_criteria(my_run_criteria));
+```
+
+After:
+
+```rust
+app.add_systems((a, b).run_if(my_run_condition));
+```
+
+#### Ambiguity detection
+
+The `ReportExecutionOrderAmbiguities` resource has been removed.
+Instead, this is configured on a per-schedule basis.
+
+```rust
+app.edit_schedule(CoreSchedule::Main, |schedule| {
+    schedule.set_build_settings(ScheduleBuildSettings {
+        ambiguity_detection: LogLevel::Warn,
+        ..default()
+    });
+})
+```
+
+#### Fixed timesteps
+
+The `FixedTimestep` run criteria has been removed, and is now handled by either a schedule or the `on_timer` / `on_fixed_timer` run conditions.
+
+Before:
+
+```rust
+app.add_stage_after(
+    CoreStage::Update,
+    FixedUpdateStage,
+    SystemStage::parallel()
+        .with_run_criteria(
+            FixedTimestep::step(0.5)
+        )
+        .with_system(fixed_update),
+);
+```
+
+After:
+
+```rust
+app.insert_resource(FixedTime::new_from_secs(0.5))

```

Comment:
```suggestion
// This will affect the update frequency of fixed time for your entire app
app.insert_resource(FixedTime::new_from_secs(0.5))

```
Not sure if needed, or the actual phrasing of it.